src/providers/documentLinkProvider.ts

```diff
@@ -7,7 +7,7 @@ import { getWorkspaceRootPath } from '../utils/workspaceUtility';
 
 export class RequestBodyDocumentLinkProvider implements DocumentLinkProvider {
 
-    private _linkPattern = /^(\<\s+)(.+)(\s*)$/g;
+    private _linkPattern = /^(\<(?:@?(?:\w+)?)?\s+)(.+)(\s*)$/g;
```

Comment:
It seems in your regex, the encoding is supported with the `<` syntax, in my option, it's only supported with the `<@` syntax

Reply:
sorted